package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/smart-transactions-controller",
-  "version": "7.0.0",
+  "version": "7.1.0",
```

Comment:
It looks like PR 301 introduced a breaking change in that SmartTransactionController now requires a new `getTransactions` option. Is this intentional? If so, then we will want to release 8.0.0 rather than 7.1.0.

``` suggestion
  "version": "8.0.0",
```